File: milestone 2/python_modules/module00/ex3/ft_plant_age.cpp
#include <iostream>

void	ft_plant_age(void)
{
	int	plant_age;

	std::cout << "Enter plant age in days: ";
	std::cin >> plant_age;
	if (plant_age > 60)
		std::cout << "Plant is ready to harvest!\n";
	else
		std::cout << "Plant needs more time to grow.\n";
}

int	main(void)
{
	ft_plant_age();
	return 0;
}

File: milestone 2/python_modules/module00/ex3/ft_plant_age.java
import java.util.Scanner;

public class ft_plant_age {
    public static void main(String[] args) {
        System.out.print("Enter plant age in days: ");
        Scanner scan = new Scanner(System.in);
        int plant_age = scan.nextInt();
        if (plant_age > 60)
            System.out.println("Plant is ready to harvest!");
        else
            System.out.println("Plant needs more time to grow.");
    }
}